main/webapp/modules/core/scripts/reconciliation/standard-service-panel.html

```diff
@@ -1,11 +1,15 @@
 <div class="recon-dialog-service-panel recon-dialog-standard-service-panel">
   <div class="grid-layout layout-normal layout-full grid-layout-for-ui"><table>
     <tr>
+      <td colspan="6" style="text-align: right;">
+        <a href="" target="_blank" bind="documentationLink"><span bind="or_proc_accessDocumentation"></span>
+        <span class="ui-icon ui-icon-link" style="display: inline-block"></span></a>
```

Comment:
We might want to have a different icon for the documentation link.

Reply:
I removed the icon!